source/install/cloud-changelog.md

```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-11-17](#release-2022-11-17)
 - [Release 2022-11-10](#release-2022-11-10)
 - [Release 2022-10-27](#release-2022-10-27)
 - [Release 2022-10-20](#release-2022-10-20)
 - [Release 2022-10-13](#release-2022-10-13)
 - [Release 2022-10-06](#release-2022-10-06)
-- [Release 2022-09-15](#release-2022-09-15)
+
+## Release 2022-11-17
+
+### Improvements
+
+#### User Interface (UI)
+ - The **Mark as Unread** option was added to the ``…`` menu for channels in the left-hand side sidebar. Pressing Alt while selecting a channel on the left-hand side now also marks the last post in the channel as unread.
+ - Channel members are now able to remove themselves from a channel via the right-hand side channel members list.
+ - Updated prepackaged Boards version to v7.5.0.
```

Comment:
@amyblais Did we not want to include the list of Board improvements as well?

Reply:
Yes, indeed, added now.